### DISCUSSION TITLE
v2/md-enhance/zh/guide/chart

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#402] [Announcements]
# v2/md-enhance/zh/guide/chart

让你 VuePress 站点中的 Markdown 文件支持图表。 此插件使用 chart.js (https://www.chartjs.org/docs/latest/) 提供相应功能。 配置 tab TS tab JS 格式 我们也支持 js 和 javascript 的代码块，你应当将导出对象赋值给 module.exports。 案例 块状图...

https://vuepress-theme-hope.github.io/v2/md-enhance/zh/guide/chart.html

**Comment by Gaobang100** [comment]
That's perfect!!!

**Comment by tsauliu** [comment]
希望可以早点加入vitepress
